INSTALL.rst

```diff
@@ -16,7 +16,17 @@ Matplotlib and its dependencies are available as wheel packages for macOS,
 Windows and Linux distributions::
 
   python -m pip install -U pip
-  python -m pip install -U matplotlib
+  python -m pip install --no-user --prefer-binary -U matplotlib
+
+The ``--prefer-binary`` flag will select the newest version of
+Matplotlib for which there is a wheel.  Remove this flag if you are
+willing to possibly compile Matplotlib from source.
```

Comment:
Do you have example of cases where this does help?  i.e. does it ever happen that there's a sdist for e.g. mpl3.m with no wheel for py3.p, but there's a py3.p wheel for an *older* mpl so that --prefer-binary would make pip pick the older version?

Reply:
When py38 came out it took a while for use to catch up.

If we move our wheels to manylinux2010, older pip's will not see them (shift versions forward as desired).

Reply:
The other place this came up recently is that some of our older versions have 32 bit wheels uploaded and skimage was having issues trying to build newer versions of mpl from source on their CI.  `--prefer-binary` fixed it enough for their needs.

Reply:
> skimage was having issues trying to build newer versions of mpl from source on their CI

This should be fixed with mpl3.2 now that we download ft by default and don't depend on libpng anymore.  (I think "mpl>=3.2 fails to build on a linux box which can otherwise build a trivial C extension" can now be considered as a bug).

> If we move our wheels to manylinux2010, older pip's will not see them (shift versions forward as desired).

True, but 1) this is a theoretical problem right now (do we even have plans to move away from manylinux1?), 2) numpy just shifted to manylinux2010 and don't seem worried about that.

Reply:
> (I think "mpl>=3.2 fails to build on a linux box which can otherwise build a trivial C extension" can now be considered as a bug)

fair, but I'm more worried about mac / windows here.

Reply:
but then the manylinux argument doesn't hold anymore :)

Reply:
fair